hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/ObjectEndpoint.java

```diff
@@ -127,11 +126,17 @@ public class ObjectEndpoint extends EndpointBase {
   private static final Logger LOG =
       LoggerFactory.getLogger(ObjectEndpoint.class);
 
+  @Context
+  private ContainerRequestContext context;
+
   @Context
   private HttpHeaders headers;
 
 
   private List<String> customizableGetHeaders = new ArrayList<>();
```

Comment:
I think `customizableGetHeaders` is no longer needed.  The same set of headers is now available as keys of `overrideQueryParameter`.

Reply:
`customizableGetHeaders` has been removed

Reply:
Thanks.  Variable declaration can be removed, too, as well as `import java.util.ArrayList`.